packages/client-sdk-nodejs/test/integration/auth.test.ts

```diff
@@ -0,0 +1,105 @@
+import {
+  GenerateApiToken,
+  MomentoErrorCode,
+  RefreshApiToken,
+} from '@gomomento/sdk-core';
+import {SetupAuthIntegrationTest} from './integration-setup';
+
+const {Momento, SessionToken} = SetupAuthIntegrationTest();
```

Comment:
i know you are just copying an existing pattern here but it's a weird pattern that I want to clean up.  Can you make these names start with a lower-case letter, and instead of `Momento` can you call this `authClient`?  I would also love to see type annotations on these two variables to make it more obvious what they are.

Reply:
For sure!